typed-protocols-examples/src/Network/TypedProtocol/ReqResp/Codec.hs

```diff
@@ -13,21 +15,26 @@ import           Text.Read (readMaybe)
 
 
 codecReqResp
-  :: forall req resp m.
-     (Monad m, Show req, Show resp, Read req, Read resp)
-  => Codec (ReqResp req resp) CodecFailure m String
+  :: forall req' resp' m.
+     (Monad m, Show req', Show resp', Read req', Read resp')
+  => Codec (ReqResp req' resp') CodecFailure m String
```

Comment:
I'd prefer if these were called `req` and `resp`.

Reply:
@coot, done!